packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -54,6 +54,13 @@ export interface ClusterProps {
    */
   readonly capacityProviders?: string[];
 
+  /**
+   * The cluster default capacity provider
```

Comment:
```suggestion
   * The cluster default capacity provider strategy. This takes the form of a list of CapacityProviderStrategy objects. 
   * For example, blah blah blah
```

Please read through the docs for capacity provider strategies, autoscaling, and managed scaling and give an example of a valid config here. 
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-capacityprovider-managedscaling.html
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-service-capacityproviderstrategyitem.html
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-capacityprovider-autoscalinggroupprovider.html